tutorials.markdown

```diff
@@ -22,7 +22,6 @@ id: tutorials
 * [Reactive Extensions: Beyond the Basics](https://www.youtube.com/watch?v=Jt-_oVQVZlQ), a presentation by Dan Lew (how to make your own operators, avoiding memory leaks in subscriptions, understanding Schedulers, hot and cold Observables, sharing Observables, the trouble with Subjects, backpressure)
 * [Reactive Programming](https://www.youtube.com/watch?v=3bAQXTVsEiQ), video of a presentation by Venkat Subramaniam
 * [ReactiveX, RxJava, and RxAndroid - where to start](http://www.andreamaglie.com/2015/rxjava-android-where-to-start/) by Andrea Maglie
-* [Building an Application from Scratch Using RxJava and Java8](http://blog.xebia.in/2015/09/01/day1-building-an-application-from-scratch-using-rxjava-and-java8/) by Shekhar Gulati
```

Comment:
This link works: http://blog.xebia.in/index.php/2015/08/31/day1-building-an-application-from-scratch-using-rxjava-and-java8/

Looks like the date changed somewhere along the way...